### ISSUE TITLE
Parse array parameter docs

### ISSUE THREAD

**Issue by smckeown** [rmccue/WP-Parser#119]
Some of the parameter docs use lists. Markdown needs to be parsed in parameter docs.

See this for reference: https://meta.trac.wordpress.org/ticket/197

**Comment by johnbillion** [comment]
Looks like this was fixed at some point, but I'm not sure where, and I can't see any test coverage for it.

Is there some post-processing happening on developer.wordpress.org? For example, [wp_list_comments()](https://developer.wordpress.org/reference/functions/wp_list_comments/) has hash notation for its `$args` parameter, yet it's displayed correctly.

**Comment by Rarst** [comment]
I... don't remember. :) Production reference does run some hacks (I had never seen myself) on top of Parser, but I doubt it's anything as extensive as dealing with those lists. A little busy with other things right now to de-dust my Parser setup and test.

**Comment by DrewAPicture** [comment]
@johnbillion @Rarst: Yes, there's still some text processing going on on the front-end to give it halfway decent formatting.

See [`param_formatting_fixup()`](https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/themes/pub/wporg-developer/inc/template-tags.php#L1186) in /inc/template-tags.php.
